public/wp-content/themes/ColoredCow/landing.php

```diff
@@ -5,9 +5,88 @@
     get_header();
 ?>
 
-<?php
-?>
- 
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col col-sm-12 col-md- col-lg-6">
+				<h2 class="about">Soiree</h2>
```

Comment:
this is a title. I don't think it has any connection with **about**, right? Use a better name.